artifacts/scripts/util.sh

```diff
@@ -851,7 +851,7 @@ update-deps-in-gomod() {
     # if dependencies exist, dep_packages is a comma separated list of {base_package}/{dep}. Eg: "k8s.io/api,k8s.io/apimachinery"
     local dep_packages=""
     if [ "$dep_count" != 0 ]; then
-      dep_packages="$(echo ${1} | tr "," "\n" | sed -e 's/:.*//' -e s/^/"${base_package}\/"/ | paste -sd "," -)"
+      dep_packages="$(echo ${1} | tr "," "\n" | sed -e 's/:.*//' -e s~^~"${base_package}/"~ | paste -sd "," -)"
```

Comment:
I think elsewhere we use comma. Tilde is used in bash. Am not so confortable to use that.

Reply:
Thanks, I updated it to be `,`. Tested with:

```console
$ echo api:master | tr "," "\n" | sed -e 's/:.*//' -e 's,^,tess.io/ebay/,'
tess.io/ebay/api
```